src/app/organizations/state/organizations.query.ts

```diff
@@ -50,4 +54,20 @@ export class OrganizationsQuery extends Query<OrganizationsState> {
     }
     return null;
   }
+
+  filterAndSortOrganizations(organizations: Array<Organization>, searchName: string, sortBy: string): Array<Organization> | null {
+    if (organizations) {
+      organizations = this.filterOrganizations(organizations, searchName);
```

Comment:
This is just me, minor and won't affect anything in this case should you change it, but my preference is not to reassign parameter values.